test/e2e/mob_test.go

```diff
@@ -71,6 +71,24 @@ func TestMachineOSBuilder(t *testing.T) {
 
 	t.Cleanup(createConfigMapForTest(t, cs))
 
+	// get the feature gates because we're gating this for now
+	featureGates, err := cs.ConfigV1Interface.FeatureGates().Get(context.TODO(), "cluster", metav1.GetOptions{})
```

Comment:
you might need to add the FGs to the cs if you want them to be initialized.

Reply:
ok- im going to clean this up in a another PR that uses this test!

Reply:
does the ClientSet itself store or manage feature gate configurations?